entries/o/optumbank.com.json

```diff
@@ -2,13 +2,10 @@
   "Optum Bank": {
     "domain": "optumbank.com",
     "url": "https://www.optumbank.com",
-    "contact": {
-      "facebook": "myOptum",
-      "twitter": "myOptum"
-    },
-    "regions": [
-      "us"
+    "tfa": [
+      "sms"
```

Comment:
You stated in your comment that SMS and email authentication is available but you've only added SMS.

Reply:
fixed